WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -211,12 +221,13 @@ extension WordPressAuthenticationManager: WordPressAuthenticatorDelegate {
 
     /// Synchronizes the specified WordPress Account.
     ///
-    func sync(credentials: WordPressCredentials, onCompletion: @escaping () -> Void) {
-        switch credentials {
-        case .wpcom(let authToken, let isJetpackLogin, _):
-            syncWPCom(authToken: authToken, isJetpackLogin: isJetpackLogin, onCompletion: onCompletion)
-        case .wporg(let username, let password, let xmlrpc, let options):
-            syncWPOrg(username: username, password: password, xmlrpc: xmlrpc, options: options, onCompletion: onCompletion)
+    func sync(credentials: AuthenticatorCredentials, onCompletion: @escaping () -> Void) {
+        if let wpcom = credentials.wpcom {
+            syncWPCom(authToken: wpcom.authToken, isJetpackLogin: wpcom.isJetpackLogin, onCompletion: onCompletion)
+        }
+
+        if let wporg = credentials.wporg {
```

Comment:
Super nitpicky, but we could compose this as an `else if` to avoid checking `wporg` if the previous conditional block was true.